### ISSUE TITLE
Add Jenkins check for equivalence to a leaf OMIM and an OMIMPS

### ISSUE THREAD

**Issue by nicolevasilevsky** [monarch-initiative/mondo#1455]
We should never be equiv to a leaf OMIM and an OMIMPS.

Related to https://github.com/monarch-initiative/mondo/pull/1430

**Comment by matentzn** [comment]
see https://github.com/monarch-initiative/mondo/issues/2277#issuecomment-725075741

**Comment by nicolevasilevsky** [comment]
I will work on these three first:

- [ ] MONDO_0008546,thanatophoric dysplasia type 1, MONDO_0014658,severe achondroplasia-developmental delay-acanthosis nigricans syndrome,,
- [ ] MONDO_0010273,"lymphoma, Hodgkin, X-linked pseudoautosomal", MONDO_0009348,classic Hodgkin lymphoma,,
- [ ]  MONDO_0013236,"pancreatic cancer, susceptibility to, 3", MONDO_0015278,familial pancreatic carcinoma,,

**Comment by nicolevasilevsky** [comment]
Summary:
1. excluded superclass from MONDO_0008546 thanatophoric dysplasia type 1 - _this did not have a source for the superclassOf assertion and it seemed like it was a mistake. `MONDO_0008546 thanatophoric dysplasia type 1` was the only child of `MONDO_0014658,severe achondroplasia-developmental delay-acanthosis nigricans syndrome`, which Chris says is a "bad smell"
2. I reclassified MONDO_0010273,"lymphoma, Hodgkin, X-linked pseudoautosomal" (and its sibling) as children of Hodgkin's lymphoma
3. I excluded the superclasses for `MONDO_0013236,"pancreatic cancer, susceptibility to, 3' (and type 2 and 4). These came from Orphanet.
